hack/README.md

```diff
@@ -69,6 +70,21 @@ These may be passed to a target as `$ make VARIABLE=value target`
 - `VERBOSITY`: (default: 1) Set global log level verbosity
 - `PULL_POLICY`: (default: IfNotPresent) Set global CDI pull policy
 
+#### Execute Functional Tests
+To Run Tests
+```
+ make cluster-up
```

Comment:
My version of this command was `KUBEVIRT_PROVIDER=k8s-1.10.4 KUBEVIRT_NUM_NODES=3 make cluster-up`. Are these variables not needed? If so what's their default?

Reply:
Technically they are not needed, as their defaults are k8s-1.10.4 and the number of nodes = 1. The environment variables allow you to override the defaults with something else like increase the number of nodes or use the open shift provider.

Reply:
@awels I didn't realize there was a default for `KUBEVIRT_PROVIDER`. Good to know and I'll close issue #331